app/scripts/arbeitsangebote/detail/arbeitsangebotedetail.html

```diff
@@ -18,8 +18,10 @@ <h3>
                 <button ng-if="arbeitsangebot.status === 'Bereit'" type="button" role="button" ng-click="aaZurueckziehen()" class="btn btn-warning"><span class="fa fa-pause"></span> <span translate>Zurückziehen</span></button>
                 <button ng-if="arbeitsangebot.status === 'Bereit'" type="button" role="button" ng-click="aaAbsagen()" class="btn btn-info"><span class="fa fa-ban"></span> <span translate>Absagen</i></span></button>
                 <button ng-if="arbeitsangebot.status === 'Bereit'" type="button" role="button" ng-click="aaAbschliessen()" class="btn btn-danger"><span class="fa fa-stop"></span> <span translate>Abschliessen <i class="fa fa-info-circle" aria-hidden="true" uib-tooltip-template="'abschliessenTooltip.html'"></i></span></button>
-                <oo-save-button model="arbeitsangebot" form="arbeitsangebotForm" entity="Arbeitsangebot" on-save="save" on-cancel="backToList" on-created="created">{{fullName() || 'Arbeitsangebot'|translate}}</oo-save-button>
-                <oo-delete-button ng-if="!arbeitsangebot.anzahlTeilnehmer || arbeitsangebot.anzahlTeilnehmer == 0" model="arbeitsangebot" entity="Arbeitsangebot" on-delete="delete" on-deleted="backToList"></oo-delete-button>
+<button ng-if="arbeitsangebot.status === 'Beendet'" type="button" role="button" ng-click="aaArchiviert()" class="btn btn-danger"><span class="fa fa-stop"></span> <span translate>Archiviert<i class="fa fa-info-circle" aria-hidden="true" uib-tooltip-template="'archiviertTooltip.html'"></i></span></button>
```

Comment:
`Archiviert` -> `Archivieren`

And there is a space missing between the text and the (i) button.

The archiviertTooltip.html is missing:

`Diesen abgeschlossenen Arbeitseinsatz als Archiviert markieren. Anschliessend ist eine Abänderung nicht mehr möglich.`

Reply:
done